src/Aquifer.API/Endpoints/Languages/List/Endpoint.cs

```diff
@@ -0,0 +1,34 @@
+using Aquifer.API.Helpers;
+using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Languages.List;
+
+public class Endpoint(AquiferDbContext dbContext) : EndpointWithoutRequest<List<Response>>
+{
+    public override void Configure()
+    {
+        Get("/languages");
+        ResponseCache(EndpointHelpers.TenMinutesInSeconds);
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.TenMinutesInSeconds));
```

Comment:
Previously this was on hour: https://github.com/BiblioNexusStudio/aquifer-server/pull/456/files#diff-eb10c1eac125190b48debcafb358d28eb6f6eb43487601c1e8013111dc0b3923L12